task7/script.js

```diff
@@ -0,0 +1,63 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク"
+}, {
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+}];
+
+// ローディング画像を表示
+const insertLoading = () => {
+  const loading = document.getElementById("div");
+  const img = new Image();
+  img.src = "loading-circle.gif";
+  img.id = "loadingImg";
+  loading.appendChild(img);
+}
+
+// ローディング画像を非表示
+const removeLoading = () => {
+  const img = document.getElementById("loadingImg");
+  img.remove();
+}
+
+// リスト作成し、htmlに差し込む
+const insertUl = (value) => {
+  listItem.forEach(function (value) {
```

Comment:
[MUST] The place that should be fixed is the 34th line
```listItem.forEach (function (value) {```
ListItem.
because,
 ListItem is specified as a global variable in the 5th to 15th lines.
At first glance it looks like it's working fine.
However, you should not run the foreach statement using the same variable name as long as you are getting the value in a global variable.
This is because it may work even if the value cannot be obtained by asynchronous communication.